module/oss.py

```diff
@@ -1,35 +1,103 @@
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see http://www.gnu.org/licenses/.
 
 DOCUMENTATION = '''
 ---
 module: oss
-short_description: manage objects in OSS.
+short_description: Create/Delete Bucket and Objects/Folder. Upload Files in OSS
 description:
-    - This module allows the user to manage OSS buckets and the objects within them. Includes support for creating and deleting both objects and buckets, retrieving objects as files or strings and generating download links. This module has a dependency on python-footmark.
+    - This module allows the user to manage OSS buckets and the objects within them. Includes support for
+      creating and deleting both objects and buckets, retrieving object keys. This module has a dependency on footmark and ossutils.
+
 common options:
-  acs_access_key:
-    description: The access key.
+  alicloud_access_key:
+    description:
+      - Aliyun Cloud access key. If not set then the value of the `ALICLOUD_ACCESS_KEY`, `ACS_ACCESS_KEY_ID`, 
+        `ACS_ACCESS_KEY` or `ECS_ACCESS_KEY` environment variable is used.
+    required: false
+    default: null
+    aliases: ['acs_access_key', 'ecs_access_key','access_key']
+  alicloud_secret_key:
+    description:
+      - Aliyun Cloud secret key. If not set then the value of the `ALICLOUD_SECRET_ACCESS_KEY`, `ACS_SECRET_ACCESS_KEY`,
+        `ACS_SECRET_KEY`, or `ECS_SECRET_KEY` environment variable is used.
     required: false
     default: null
-    aliases: []
-  acs_secret_access_key:
-    description: The access secret key.
+    aliases: ['acs_secret_access_key', 'ecs_secret_key','secret_key']
+  alicloud_region:
+    description:
+      - The Aliyun Cloud region to use. If not specified then the value of the `ALICLOUD_REGION`, `ACS_REGION`, 
+        `ACS_DEFAULT_REGION` or `ECS_REGION` environment variable, if any, is used.
     required: false
     default: null
-    aliases: []
+    aliases: ['acs_region', 'ecs_region', 'region']
   mode:
     description:
-      - Switches the module behaviour between put (upload), get (download), geturl (return download url), getstr (download object as string), list (list keys), create (bucket), delete (bucket), and delobj (delete object).
+      - Switches the module behaviour between create (bucket), delete (bucket), put (upload), put_folder
+        (create folder), list (list keys) and delobj (delete object).
     required: true
-    choices: ['get', 'put', 'delete', 'create', 'geturl', 'getstr', 'delobj', 'list']
+    choices: ['create', 'delete', 'put', 'put_folder', 'list', 'delobj']
+
+
+function: create bucket

```

Comment:
We have checked in API documentation at http://aliyun-oss-python-sdk.readthedocs.io/en/stable/api.html#oss2.Bucket.put_bucket_acl but did not find 'secure' parameter . Please let us know if we are missing anything.